src/acceleration/config/AccelerationConfiguration.cpp

```diff
@@ -263,11 +266,13 @@ void AccelerationConfiguration::xmlEndTagCallback(
         _preconditioner = PtrPreconditioner(new ResidualSumPreconditioner(_config.precond_nbNonConstTWindows));
       } else {
         // no preconditioner defined
+        /*if ()
         std::vector<double> factors;
         for (int id = 0; id < static_cast<int>(_config.dataIDs.size()); ++id) {
           factors.push_back(1.0);
         }
-        _preconditioner = PtrPreconditioner(new ConstantPreconditioner(factors));
+        _preconditioner = PtrPreconditioner(new ConstantPreconditioner(factors));*/
+        _preconditioner = PtrPreconditioner(new ResidualSumPreconditioner(_defaultValuesIQNILS.precond_nbNonConstTWindows));
```

Comment:
Please don't comment lines out, but directly delete them. Will only confuse someone in three years. "Why is this commented out, should I comment it in again?"